brave/browser-laptop
# Reorganize about pages and less files

**Requires #10951 for buttons on about:autofill.**


Fixes #10711
Closes #8056
Closes #10700
Closes #10712
Closes #10880
Closes #11120

- Add less/about/common.less to the about js files
- Add globalStyles.spacing.aboutPageMargin
- Add the MPL license header
- Remove import from the less files
- Remove button.less from common.less
- Remove require of the less files imported in common.less to avoid duplicates
- Move the height:100% property from common.less to preferences.less

Please have a look at https://github.com/brave/browser-laptop/issues/10880 why I thought this reorganization is important, thanks.

# Test Plans

## 1st commit

Closes #10700
Closes #10880
Addresses #8056

Test Plan 1: `npm run unittest`

Test Plan 2:
1. Open `about:about`
2. Open all linked about pages
3. Make sure they are properly styled
4. Open https://webtorrent.io/free-torrents
5. Open a torrent link
6. Make sure the webtorrent viewer is properly styled

Test Plan 3:
1. Open `https://expired.badssl.com` (for certerror.js)
2. Open `https://brave.rules` (for errorPage.js)
3. Open `about:preferences` (for preferences.js)
4. Open `http://downloadme.org` (for safebrowsing.js)
5. Make sure the all error pages are rendered
6. Make sure all error pages do not have the black background color

Test Plan 4:
1. Open `about:brave`
2. Change the window size
3. Make sure the scroll bar does not appear

## 2nd commit

Fixes #10711
Closes #10712

Test Plan:
1. Open about:brave
2. Make sure the layout is not broken

## 3rd commit

Closes #8056

Test Plan:
1. Open about:about
2. Make sure the page is properly rendered

## 4th commit 

Closes #11120

Test Plan:
1. Open about:about
2. Open about:adblock
3. Open about:autofill
4. Open about:brave
5. Open about:downloads
6. Open about:extensions
7. Make sure the section titles are styled in the same way

----


Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
